p2p/Cargo.toml

```diff
@@ -19,6 +19,7 @@ description = """
 chacha20poly1305 = "0.7"
 ed25519-dalek = "1"
 eyre = "0.6"
+anyhow = "1.0"
```

Comment:
Dependencies are ordered alphanumerically.

Reply:
Why the introduction of the extra dependency?

Reply:
Can you also explain why the `p2p` crate now needs to depend on `anyhow`?

Reply:
Oh, sorry, I forget delete this crate. The current is can delete.